packages/dds/merge-tree/src/client.ts

```diff
@@ -255,9 +256,26 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 	 * @param start - The inclusive start of the range to obliterate
 	 * @param end - The exclusive end of the range to obliterate
 	 */
-	// eslint-disable-next-line import/no-deprecated
-	public obliterateRangeLocal(start: number, end: number): IMergeTreeObliterateMsg {
-		const obliterateOp = createObliterateRangeOp(start, end);
+	public obliterateRangeLocal(
+		start: SequencePlace,
```

Comment:
as we talked about in chat, can we make this `number| InteriorSequencePlace`

Reply:
i think this should be reverted for now, and eventually we can make it be interior sequence place